src/index.mjs

```diff
@@ -58,6 +62,47 @@ export function respondCORS() {
   });
 }
 
+function randomZeroOrOne() {
+  return (Math.random() >= 0.5) ? 1 : 0;
+}
+
+export function getOtherPackageRegistry(regName) {
+  return regName === PACKAGE_REGISTRIES[1] ? PACKAGE_REGISTRIES[0] : PACKAGE_REGISTRIES[1];
```

Comment:
```suggestion
  return PACKAGE_REGISTRIES[(PACKAGE_REGISTRIES.indexOf("three") + 1) % PACKAGE_REGISTRIES.length]
```

what if, no `if`?

Reply:
I used slightly different logic, but now it computes another value based on the length of the list, so it will also work if there are more than 2 items.